hal/src/main/native/cpp/jni/HALUtil.cpp

```diff
@@ -68,7 +69,9 @@ static const JClassInit classes[] = {
     {"edu/wpi/first/hal/CANStreamMessage", &canStreamMessageCls},
     {"edu/wpi/first/hal/HALValue", &halValueCls},
     {"edu/wpi/first/hal/DMAJNISample$BaseStore", &baseStoreCls},
-    {"edu/wpi/first/hal/REVPHVersion", &revPHVersionCls}};
+    {"edu/wpi/first/hal/REVPHVersion", &revPHVersionCls},
+    {"edu/wpi/first/hal/can/CANStreamOverflowException",
+     &canStreamOverflowExCls}};
```

Comment:
Is there a reason this was added to `classes` instead of `exceptions`?

Reply:
Because exceptions expects there to be a string taking constructor, which this doesn't have.